### PR TITLE
mtx: add command utils

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/mtx/utils/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -0,0 +1,1420 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable func-names */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+
+import {
```
Comment (DarkPark) [inline src/mtx/utils/CommandBinaryBuffer.ts:7]:
```suggestion
import * as types ...
```

File: src/mtx/message/downlink.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ *
+ * @example
+ * ```js
+ * import * as message from 'jooby-codec/analog/message/downlink';
+ * import * as downlinkCommands from 'jooby-codec/analog/commands/downlink';
+ * import getHexFromBytes from 'jooby-codec/utils/getHexFromBytes.js';
+ *
+ * const commands = [
+ *     {
+ *         id: downlinkCommands.correctTime2000.id,
```
Comment (DarkPark) [inline src/mtx/message/downlink.ts:12]: wrong example command

File: src/mtx/utils/CommandBinaryBuffer.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/mtx/utils/CommandBinaryBuffer.ts:7]: fixed

File: src/mtx/utils/command.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+export type TCommandExamples = Record<TExampleName, TCommand>;
+
+/**
+ * Command public interface exported from its module.
+ */
+export interface ICommandImplementation {
+    id: types.TCommandId,
+    name: types.TCommandName,
+    maxSize: number,
+    accessLevel: number,
```
Comment (DarkPark) [inline src/mtx/utils/command.ts:45]:
```suggestion
    accessLevel: types.TAccessLevel,
```
?

Comment (DeadByElpy) [reply src/mtx/utils/command.ts]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**